integr8ly/cloud-resource-operator
# MGDAPI-3319 - correctly populate manifest from base

## Overview

Jira: [MGDAPI-3319](https://issues.redhat.com/browse/MGDAPI-3319)


## Changes

- Using operator-sdk variable instead of calling directly, and updated to 1.14.0
- Now updating bases file with `cloud-resource-operator.v{VERSION}` so that new releases populated correctly
- Removal of manually updating `installModes` as they're passed from base file
- Removed trailing close-quote from base64data icon image in base file

## Verification

- Create new release
- Verify contents are expected (match v0.34.0)


## Checklist
